ArduCopter/mode_stabilize.cpp

```diff
@@ -54,12 +54,34 @@ void ModeStabilize::run()
         break;
     }
 
-    // call attitude controller
-    if ((auto_yaw.mode() != AUTO_YAW_FIXED)
-        || (target_yaw_rate > 200) || (target_yaw_rate < -200)) {
+    AP_Mount *mount = AP::mount();
```

Comment:
Please remove the mods to `mode_stabilize.cpp`.  `poshold` is fine, but stabilize is our backup mode, so if the gimbal is controlling the yaw we could get into trouble.